tests/Earthfile

```diff
@@ -610,6 +610,26 @@ wildcard-build-pwd:
     RUN cat earthly.output | acbgrep 'hello world 3'
     RUN cat earthly.output | acbgrep -v 'not-test run'
 
+wildcard-build-pwd-rel:
+    COPY wildcard-build subdir
+    COPY wildcard-build.earth subdir/Earthfile
+    DO +RUN_EARTHLY --earthfile=wildcard-build.earth --target=./subdir+wildcard-build-pwd
+    RUN cat earthly.output | acbgrep 'hello world 1'
+    RUN cat earthly.output | acbgrep 'hello world 2'
+    RUN cat earthly.output | acbgrep 'hello world 3'
+    RUN cat earthly.output | acbgrep -v 'not-test run'
+
+wildcard-build-rel-dir:
+    WORKDIR /usr/local
```

Comment:
is this intentional? can we pick a different directory instead, like `/root/special/test/directory`? Using `/usr/local` makes this test feel like this test is installing something on the container's system.

Reply:
I chose an arbitrary system directory to help test the relative path resolution. I'd rather not create a long-winded custom directory. What about `/home/root`?

Reply:
`/home/root` seems a bit odd too, since typically root is located under `/root`, `/root/some-test-dir` feels more in line with linux file paths.

Reply:
Yes, thanks. I meant the home dir for root. The difference slipped my mind.